.gitlab/build_and_test.sh

```diff
@@ -14,7 +14,7 @@ mkdir $BUILD_DIR
 cd $BUILD_DIR
 
 cmake -C ../host-configs/llnl/$SYS_TYPE/$HOST_CONFIG ../tests/internal
-make -j8
+make VERBOSE=1 -j8
```

Comment:
Does this need to be removed?

Reply:
it's good to be able to just look at the log in the CI